content/hi/contribute/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: कैसे योगदान करें
+toc_hide: true
+menu:
+  main:
+    weight: 10
+    pre: <i class='fas fa-pen-square'></i>
+---
+
+क्लाउड नेटिव शब्दावली के लिए सभी सामग्री [इस गिटहब रेपो](https://github.com/cncf/glossary) में संग्रहीत है| आपको [मुद्दों, issues](https://github.com/cncf/glossary/issues), [PRs](https://github.com/cncf/glossary/pulls), और [चर्चाओं, discussions](https://github.com/cncf/glossary/discussions) की एक सूची मिलेगी शब्दावली के बारे में।
+
+## सामान्य दिशानिर्देश
+
+शब्दावली प्रविष्टि में विशिष्ट परिवर्तनों का प्रस्ताव करने के लिए, अपनी शाखा(branch) में उस प्रविष्टि(entry) को संपादित करें और पुल रिक्वेस्ट(pull request) जारी करें। यह अनुरोध करने के लिए कि किसी प्रविष्टि को स्पष्ट किया जाए, अद्यतन किया जाए, या उस पर पुनर्विचार किया जाए, आप वैकल्पिक रूप से कोई समस्या(issue) खोल सकते हैं। शब्दावली में एक नई प्रविष्टि का प्रस्ताव करने के लिए, या तो एक मुद्दा बनाएं या, यदि आपने परिभाषा का मसौदा तैयार किया है, तो अपनी शाखा में प्रविष्टि जोड़ें और एक पुल रिक्वेस्ट बनाएं।
+
+## मुद्दे
+
+कृपया इसमें शामिल हों और [ओपन इश्यूज](https://github.com/cncf/glossary/issues) की समीक्षा करके और उन्हें हल करने के लिए पुल रिक्वेस्ट करके हमारी मदद करें। सबसे आसान लोगों को "गुड फर्स्ट इशू (good first issue)" या "हेल्प वांटेड (help wanted)" टैग के साथ चिह्नित किया गया है। वह चुनें जिसे पहले से किसी को असाइन नहीं किया गया है|
+
+अन्य जिन्हें अभी तक असाइन नहीं किया गया है, हो सकता है कि किसी के द्वारा "दावा" किया गया हो। इसके बारे में अधिक जानने के लिए इस मुद्दे पर क्लिक करें।
+
+आप शब्दावली में किसी भी पृष्ठ के दाहिने साइडबार में "Report issue" पर क्लिक करके एक नया मुद्दा सबमिट कर सकते हैं।
+
+## एक टर्म अपडेट करना (उर्फ पीआर सबमिट करना)
+शब्दावली प्रविष्टि को अद्यतन करने के लिए इन चरणों का पालन करें:
+1. उस टर्म पर नेविगेट करें जिसे आप संपादित करना चाहते हैं
+2. दाएँ साइडबार में "Edit this page" लिंक पर क्लिक करें
+3. रिपॉजिटरी का अपना फोर्क बनाएं
+4. कंटेंट में अपने परिवर्तन करें
+5. पुल रिक्वेस्ट बनाएं
+
+कृपया अपने पुल अनुरोधों को उचित रूप से शीर्षक दें, संक्षेप में बताएं कि आपकी कमिटस किस बारे में है। साथ ही, कृपया प्रत्येक परिवर्तन के लिए अलग-अलग पुल अनुरोध करें क्योंकि इससे स्व-निहित इकाइयों में चर्चा और चरवाहा अद्यतन करना आसान हो जाता है। तृतीय-पक्ष उपयोगों के लिंक प्रदान करें जो आपकी समस्या या पुल रिक्वेस्ट का समर्थन करते हैं।
```

Comment:
Shouldn't we keep `pull requests` as `पुल रिक्वेस्ट` as it is a proper technical term.

Reply:
> @anubha-v-ardhan I was about to add the style guide as well, but are we translating its content or keeping that in English

I would recommend one page per PR (This PR can go with these pages) but can we do one PR per page. It will help us to keep track of progress better and will be faster for review process. It will also allow new contributors to work on one page and start their open-source journey

Reply:
Agreed that its a technical term but I think पुल अनुरोध is fine as it conveys the same meaning. I believe we can go ahead unless its the same meaning.
We can have pull request parenthesis if you wish.

Reply:
I guess pull request in parenthesis next to the word will do.